src/AutomatedCar/GameBase.cs

```diff
@@ -5,7 +5,7 @@ namespace AutomatedCar
 
     public abstract class GameBase
     {
-        public const int TicksPerSecond = 60;
+        public const int TicksPerSecond = 30;
```

Comment:
Ezzel nem értek egyet, mert a GameBase őse a VFB-nek és a Game-nek is, így kihathat a többi csapatra úgy, hogy erről nem is tudnak.

@SzFMV2020-Osz/company Okozhat gondot bárkinek ha ez itt át van írva?